README.md

```diff
@@ -83,6 +83,19 @@ of options:
  * [Libvirt][libvirt]
  * [Cockpit][cockpit]
 
+## Submitting patches
+
+When sending patches to the project, the submitter is required to certify that
+they have the legal right to submit the code. This is achieved by adding a line
+
+    Signed-off-by: Real Name <email@address.com>
```

Comment:
It looks like it does make sense, even if Kubernetes is not using it (yet).

The Kernel uses it, as well as OpenStack and oVirt, to name a few.
And docker also introduced with some reasoning: https://blog.docker.com/2014/01/docker-code-contributions-require-developer-certificate-of-origin/
According to that post, the DCO is an alternative to a contributer license agreement (CLA) which we've actually discussed before. (This is probaly why K8s does not have a DCO, because they have an CLA).

Long story short, there is no drawback in requiring the signed off part, and it seem that we'll have a better standing on the legal side.